pyaedt/icepak.py

```diff
@@ -768,9 +768,7 @@ def create_network_block(
         >>> block.props["Nodes"]["Internal"][0]
         '2W'
         """
-        warnings.warn(
-            "This method is deprecated in 0.6.27. Please use create_two_resistor_network_block", DeprecationWarning
-        )
+        warnings.warn("This method is deprecated in 0.6.27. Use create_two_resistor_network_block", DeprecationWarning)
```

Comment:
Can we use the suggested format in the second sentence of the warning: Use the ``create_two_resistor_network_block()`` method.

I ask because I'm not a programmer and do not know what the subsequent comma and DeprecationWarning does!